### ISSUE TITLE
"is" and "image"

### ISSUE THREAD

**Issue by BlazeGod88** [ShadowMario/FNF-PsychEngine#2127]
### Describe your bug here.

In the source of "StoryMenuState" it appear this:
![Sin título](https://user-images.githubusercontent.com/96097475/146060709-6874054b-edab-4c49-ac9c-cf3ac6c487dd.png)
I don´t know how to solve it

### Are you modding a build from source or with Lua?

Source

### What is your build target?

Windows

### Did you edit anything in this build? If so, mention or summarize your changes.

no

**Comment by MayoALT** [comment]
update your haxe

**Comment by tposejank** [comment]
duplicate of #2053 #2049 #2038 #2028 #2027 #2026 #2024 #2067
and similar, ***update your haxe.***

**Comment by BlazeGod88** [comment]
i did it but it only solucionate the "image"

![image](https://user-images.githubusercontent.com/96097475/146067975-22ac97c6-0223-41be-9f76-34104d700f1a.png)

**Comment by HiroMizuki** [comment]
> i did it but it only solucionate the "image"
> 
> ![image](https://user-images.githubusercontent.com/96097475/146067975-22ac97c6-0223-41be-9f76-34104d700f1a.png)

did u installed haxe 4.2.4 by setup?

**Comment by BlazeGod88** [comment]
yes i have installed it

**Comment by Lite6G** [comment]
I make this and it works, maybe for u too
![unknown](https://user-images.githubusercontent.com/96150137/146086667-465f2e1b-d053-4146-8c23-56c0df0ccc1f.png)

**Comment by BlazeGod88** [comment]
thx i will see

**Comment by BlazeGod88** [comment]
i do all things to do and it appears this text

C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 : Expected TypeDefinition but got {name: _Event_Void_Void, params: [{name: T}], pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47), fields: [{name: canceled, doc: 
C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 :                 Whether the event was canceled during the previous dispatch
C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 :         , pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:31: characters 2-42), access: [APublic], kind: FProp(default,null,TPath({name: <...>, params: <...>, pos: <...>, pack: <...>}),null), meta: [], name_pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:31: characters 13-21)},{name: __repeat, doc: null, 
pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:34: characters 29-61), access: [APublic], kind: FVar(TPath({name: <...>, params: <...>, pos: <...>, pack: <...>}),null), meta: [{name: :noCompletion, params: [], pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:34: characters 2-16)},{name: :dox, params: [<...>], pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:34: characters 17-22)}], name_pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:34: characters 40-48)},{name: __priorities, doc: null, pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:36: characters 17-53), access: [APrivate], kind: FVar(TPath({name: <...>, params: <...>, pos: <...>, pack: <...>}),null), meta: [{name: :noCompletion, params: [], pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:36: characters 2-16)}], name_pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:36: characters 29-41)},{name: new, doc:
C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 :                 Creates a new Event instance
C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 :         , pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:41: lines 41-49), access: [APublic], kind: FFun({ret: null, params: [], expr: {pos: <...>, expr: <...>}, args: []}), meta: 
[], name_pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:41: characters 18-21)},{name: add, doc:
C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 :                 Adds a new event listener
C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 :                 @param  listener        A callback that 
matches the signature of the event
C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 :                 @param  once    Whether to receive an event dispatch only once, or each time it is fired
C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 :                 @param  priority        The priority for this listener, a higher priority will be dispatched sooner
C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 :         , pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:57: lines 57-75), access: [APublic], kind: FFun({ret: TPath(<...>), params: [], expr: {pos: <...>, expr: <...>}, args: [<...>,<...>,<...>]}), meta: [], name_pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:57: characters 18-21)},{name: cancel, doc: 
C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 :                 Marks the event as canceled, and stops the current event dispatch
C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 :         , pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:80: lines 80-83), access: [APublic], kind: FFun({ret: TPath(<...>), params: [], expr: {pos: <...>, expr: <...>}, args: []}), meta: [], name_pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:80: characters 18-24)},{name: has, doc:
C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 :                 Checks whether a callback is a listener 
to this event
C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 :                 @param  listener        A callback that 
matches the signature of the event
C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 :                 @return Whether the callback is a listener
C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 :         , pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:126: lines 126-136), access: [APublic], kind: FFun({ret: TPath(<...>), params: [], expr: {pos: <...>, expr: <...>}, args: [<...>]}), meta: [], name_pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:126: characters 18-21)},{name: remove, doc:
C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 :                 Removes an event listener
C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 :                 @param  listener        A callback that 
matches the signature of the event
C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 :         , pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:142: lines 142-157), access: [APublic], kind: FFun({ret: TPath(<...>), params: [], expr: {pos: <...>, expr: <...>}, args: [<...>]}), meta: [], name_pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:142: characters 18-24)},{name: removeAll, doc:      
C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 :                 Removes all event listeners
C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 :         , pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:162: lines 162-171), access: [APublic], kind: FFun({ret: TPath(<...>), params: [], expr: {pos: <...>, expr: <...>}, args: [rc/lime/media/AudioSource.hx:14: characters 30-47)},{name: :no,0/src/lime/app/Event.hx:162: characters 18-27)},{name: __listeners, pos: #pos(C:/Completion, pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/l14: characters 30-47), access: [APublic], kind: FVar(TPath({name: <...>, params: <ime/media/AudioSource.hx:14: characters 30-47)},{name: :fileXmoolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47), acl, params: [{pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/pr: {pos: <...>, expr: <...>}, args: []})}], pack: [lime,app], kind: TDClass(null,lime/_internal/macros/EventMacro.hx:153: characters 48-69), exC:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/_internal/macros/EventMacro.hx:148: chpr: EConst(<...>)}], pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,it/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47)},{name:9,0/src/lime/media/AudioSource.hx:14: characters 30-47)},{namerc/lime/media/AudioSource.hx:14: characters 30-47)},{name: :fileXml, params: [{pos: :noDebug, pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lacros/EventMacro.hx:153: characters 48-69), expr: EConst(<...>)}], pos: #pos(C:/Haime/media/AudioSource.hx:14: characters 30-47)}]} (see dump/de: characters 30-47)},{name: :noDebug, pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0coding_error.txt for details)  

I really don´t know what this is

**Comment by Lite6G** [comment]
> i do all things to do and it appears this text
> 
> C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 : Expected TypeDefinition but got {name: _Event_Void_Void, params: [{name: T}], pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47), fields: [{name: canceled, doc: C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 : Whether the event was canceled during the previous dispatch C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 : , pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:31: characters 2-42), access: [APublic], kind: FProp(default,null,TPath({name: <...>, params: <...>, pos: <...>, pack: <...>}),null), meta: [], name_pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:31: characters 13-21)},{name: __repeat, doc: null, pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:34: characters 29-61), access: [APublic], kind: FVar(TPath({name: <...>, params: <...>, pos: <...>, pack: <...>}),null), meta: [{name: :noCompletion, params: [], pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:34: characters 2-16)},{name: :dox, params: [<...>], pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:34: characters 17-22)}], name_pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:34: characters 40-48)},{name: __priorities, doc: null, pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:36: characters 17-53), access: [APrivate], kind: FVar(TPath({name: <...>, params: <...>, pos: <...>, pack: <...>}),null), meta: [{name: :noCompletion, params: [], pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:36: characters 2-16)}], name_pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:36: characters 29-41)},{name: new, doc: C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 : Creates a new Event instance C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 : , pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:41: lines 41-49), access: [APublic], kind: FFun({ret: null, params: [], expr: {pos: <...>, expr: <...>}, args: []}), meta: [], name_pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:41: characters 18-21)},{name: add, doc: C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 : Adds a new event listener C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 : @param listener A callback that matches the signature of the event C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 : @param once Whether to receive an event dispatch only once, or each time it is fired C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 : @param priority The priority for this listener, a higher priority will be dispatched sooner C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 : , pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:57: lines 57-75), access: [APublic], kind: FFun({ret: TPath(<...>), params: [], expr: {pos: <...>, expr: <...>}, args: [<...>,<...>,<...>]}), meta: [], name_pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:57: characters 18-21)},{name: cancel, doc: C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 : Marks the event as canceled, and stops the current event dispatch C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 : , pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:80: lines 80-83), access: [APublic], kind: FFun({ret: TPath(<...>), params: [], expr: {pos: <...>, expr: <...>}, args: []}), meta: [], name_pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:80: characters 18-24)},{name: has, doc: C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 : Checks whether a callback is a listener to this event C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 : @param listener A callback that matches the signature of the event C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 : @return Whether the callback is a listener C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 : , pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:126: lines 126-136), access: [APublic], kind: FFun({ret: TPath(<...>), params: [], expr: {pos: <...>, expr: <...>}, args: [<...>]}), meta: [], name_pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:126: characters 18-21)},{name: remove, doc: C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 : Removes an event listener C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 : @param listener A callback that matches the signature of the event C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 : , pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:142: lines 142-157), access: [APublic], kind: FFun({ret: TPath(<...>), params: [], expr: {pos: <...>, expr: <...>}, args: [<...>]}), meta: [], name_pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:142: characters 18-24)},{name: removeAll, doc: C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 : Removes all event listeners C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 : , pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:162: lines 162-171), access: [APublic], kind: FFun({ret: TPath(<...>), params: [], expr: {pos: <...>, expr: <...>}, args: [rc/lime/media/AudioSource.hx:14: characters 30-47)},{name: :no,0/src/lime/app/Event.hx:162: characters 18-27)},{name: __listeners, pos: #pos(C:/Completion, pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/l14: characters 30-47), access: [APublic], kind: FVar(TPath({name: <...>, params: <ime/media/AudioSource.hx:14: characters 30-47)},{name: :fileXmoolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47), acl, params: [{pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/pr: {pos: <...>, expr: <...>}, args: []})}], pack: [lime,app], kind: TDClass(null,lime/_internal/macros/EventMacro.hx:153: characters 48-69), exC:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/_internal/macros/EventMacro.hx:148: chpr: EConst(<...>)}], pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,it/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47)},{name:9,0/src/lime/media/AudioSource.hx:14: characters 30-47)},{namerc/lime/media/AudioSource.hx:14: characters 30-47)},{name: :fileXml, params: [{pos: :noDebug, pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lacros/EventMacro.hx:153: characters 48-69), expr: EConst(<...>)}], pos: #pos(C:/Haime/media/AudioSource.hx:14: characters 30-47)}]} (see dump/de: characters 30-47)},{name: :noDebug, pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0coding_error.txt for details)
> 
> I really don´t know what this is

i think is compiling

**Comment by Lite6G** [comment]
> i do all things to do and it appears this text
> 
> C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 : Expected TypeDefinition but got {name: _Event_Void_Void, params: [{name: T}], pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47), fields: [{name: canceled, doc: C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 : Whether the event was canceled during the previous dispatch C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 : , pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:31: characters 2-42), access: [APublic], kind: FProp(default,null,TPath({name: <...>, params: <...>, pos: <...>, pack: <...>}),null), meta: [], name_pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:31: characters 13-21)},{name: __repeat, doc: null, pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:34: characters 29-61), access: [APublic], kind: FVar(TPath({name: <...>, params: <...>, pos: <...>, pack: <...>}),null), meta: [{name: :noCompletion, params: [], pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:34: characters 2-16)},{name: :dox, params: [<...>], pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:34: characters 17-22)}], name_pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:34: characters 40-48)},{name: __priorities, doc: null, pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:36: characters 17-53), access: [APrivate], kind: FVar(TPath({name: <...>, params: <...>, pos: <...>, pack: <...>}),null), meta: [{name: :noCompletion, params: [], pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:36: characters 2-16)}], name_pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:36: characters 29-41)},{name: new, doc: C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 : Creates a new Event instance C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 : , pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:41: lines 41-49), access: [APublic], kind: FFun({ret: null, params: [], expr: {pos: <...>, expr: <...>}, args: []}), meta: [], name_pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:41: characters 18-21)},{name: add, doc: C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 : Adds a new event listener C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 : @param listener A callback that matches the signature of the event C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 : @param once Whether to receive an event dispatch only once, or each time it is fired C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 : @param priority The priority for this listener, a higher priority will be dispatched sooner C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 : , pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:57: lines 57-75), access: [APublic], kind: FFun({ret: TPath(<...>), params: [], expr: {pos: <...>, expr: <...>}, args: [<...>,<...>,<...>]}), meta: [], name_pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:57: characters 18-21)},{name: cancel, doc: C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 : Marks the event as canceled, and stops the current event dispatch C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 : , pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:80: lines 80-83), access: [APublic], kind: FFun({ret: TPath(<...>), params: [], expr: {pos: <...>, expr: <...>}, args: []}), meta: [], name_pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:80: characters 18-24)},{name: has, doc: C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 : Checks whether a callback is a listener to this event C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 : @param listener A callback that matches the signature of the event C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 : @return Whether the callback is a listener C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 : , pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:126: lines 126-136), access: [APublic], kind: FFun({ret: TPath(<...>), params: [], expr: {pos: <...>, expr: <...>}, args: [<...>]}), meta: [], name_pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:126: characters 18-21)},{name: remove, doc: C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 : Removes an event listener C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 : @param listener A callback that matches the signature of the event C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 : , pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:142: lines 142-157), access: [APublic], kind: FFun({ret: TPath(<...>), params: [], expr: {pos: <...>, expr: <...>}, args: [<...>]}), meta: [], name_pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:142: characters 18-24)},{name: removeAll, doc: C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 : Removes all event listeners C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47 : , pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/app/Event.hx:162: lines 162-171), access: [APublic], kind: FFun({ret: TPath(<...>), params: [], expr: {pos: <...>, expr: <...>}, args: [rc/lime/media/AudioSource.hx:14: characters 30-47)},{name: :no,0/src/lime/app/Event.hx:162: characters 18-27)},{name: __listeners, pos: #pos(C:/Completion, pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/l14: characters 30-47), access: [APublic], kind: FVar(TPath({name: <...>, params: <ime/media/AudioSource.hx:14: characters 30-47)},{name: :fileXmoolkit/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47), acl, params: [{pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/pr: {pos: <...>, expr: <...>}, args: []})}], pack: [lime,app], kind: TDClass(null,lime/_internal/macros/EventMacro.hx:153: characters 48-69), exC:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/_internal/macros/EventMacro.hx:148: chpr: EConst(<...>)}], pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,it/haxe/lib/lime/7,9,0/src/lime/media/AudioSource.hx:14: characters 30-47)},{name:9,0/src/lime/media/AudioSource.hx:14: characters 30-47)},{namerc/lime/media/AudioSource.hx:14: characters 30-47)},{name: :fileXml, params: [{pos: :noDebug, pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lacros/EventMacro.hx:153: characters 48-69), expr: EConst(<...>)}], pos: #pos(C:/Haime/media/AudioSource.hx:14: characters 30-47)}]} (see dump/de: characters 30-47)},{name: :noDebug, pos: #pos(C:/HaxeToolkit/haxe/lib/lime/7,9,0coding_error.txt for details)
> 
> I really don´t know what this is

i dont know either

**Comment by HiroMizuki** [comment]
welp, maybe u need to redo the Flixel installation, i did it and it worked, idk

**Comment by HiroMizuki** [comment]
if u don't get it to work, maybe try to uninstall and install all the needed things again, and following fnf building instructions

**Comment by ShadowMario** [comment]
![Screenshot_1](https://user-images.githubusercontent.com/44785097/146123980-924ffd44-433b-4894-a9bd-8a994b066a3f.png)
You should update your repo before posting issues...
